### PR TITLE
Add sqllab logging and new tab for access requests.

### PR DESCRIPTION
**Pull request by timifasubaa**
This PR adds 

https://github.com/apache/incubator-superset/pull/5862

https://github.com/apache/incubator-superset/pull/5859

https://github.com/apache/incubator-superset/pull/5844

@airbnb/superset-fork-maintainers

### CODE REVIEW

File: superset/assets/src/components/StackTraceMessage.jsx [hunk 1]
```diff
@@ -38,7 +38,13 @@ class StackTraceMessage extends React.PureComponent {
         >
           {this.props.message}
           {this.hasLink() &&
-          <a href={this.props.queryResponse.link}> (Request Access) </a>
+          <a
+            href={this.props.queryResponse.link}
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            (Request Access)
```
Comment (williaster) [inline superset/assets/src/components/StackTraceMessage.jsx]: in the UI does this have correct spacing? i.e., is it

`my message(Request Access)` (bad, no space after message) or
`my message (Request Access)` (good)

Comment (williaster) [reply superset/assets/src/components/StackTraceMessage.jsx]: if not I'd suggest changing the line 

`(Request Access)` to
`{' (Request Access)'}`

Comment (williaster) [reply superset/assets/src/components/StackTraceMessage.jsx]: (I guess this should be done in apache, sorry I missed it there)

Comment (timifasubaa) [reply superset/assets/src/components/StackTraceMessage.jsx]: Thanks Chris, I will send out a follow-up PR on open source.

**Pull request merged**